examples/dev-mashup/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "local-mashup",
+  "version": "1.0.0",
+  "description": "Simple local mashup",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "peerDependencies": {
+    "@nebula.js/stardust": "1.0.2-alpha.0"
```

Comment:
why is this a peer dependencies?
I think it should be a normal (or maybe dev) dependency

Reply:
yea true, probably doesn't matter as it will be added anyway. I'll change version as well to latest.

Reply:
It should be peerDependencies since we use sn-bar-chart and sn-line-chart in this example and sn-bar-chart and sn-line-chart also use stardust.
I tested moving stardust to dependencies but I got problems!